x-pack/plugins/ml/server/models/data_recognizer/modules/apache_ecs/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "apache_ecs",
-  "title": "Apache access logs",
-  "description": "Find unusual activity in HTTP access logs from filebeat (ECS).",
+  "title": "Apache access logs (filebeat)",
```

Comment:
```suggestion
  "title": "Apache access logs (Filebeat)",
```

Reply:
Done in https://github.com/elastic/kibana/pull/97801/commits/8c3e500523ea5a4fff63402dd13a166a0a048c48